paddle/phi/api/yaml/generator/api_base.py

```diff
@@ -709,11 +722,16 @@ def gen_kernel_code(self, kernel_name, code_indent, inplace_flag=False):
 {code_indent}      "{kernel_name}", {{kernel_backend, kernel_layout, kernel_data_type}}{cudnn_args});
 {code_indent}  const auto& kernel = kernel_result.kernel;
 {code_indent}  VLOG(6) << "{kernel_name} kernel: " << kernel;
-
 {code_indent}  auto* dev_ctx = GetDeviceContextByBackend(kernel_result.has_fallback_cpu ? Backend::CPU : kernel_backend);
+{code_indent}  std::map<std::string, std::vector<phi::DDim>> input_shapes;
```

Comment:
No extra data structure construct to make sure performance will not be affected by it

Reply:
Done